pkg/apis/core/validation/validation_test.go

```diff
@@ -21780,8 +21780,8 @@ func TestValidateHostUsers(t *testing.T) {
 			}},
 		},
 	}, {
-		name:    "hostUsers=false - unsupported volume",
-		success: false,
+		name:    "hostUsers=false - stateful volume",
+		success: true,
```

Comment:
nit, do we want to update the comment below that says: `		// It should ignore unsupported volumes with hostUsers=true.` ? There are no unsupported volume types now

(line 21798)

Reply:
yes, thanks for noticing it.  I'll push a fixed version